book/website/project-design/filenaming.md

```diff
@@ -25,7 +25,7 @@ File names should be:
 
 Before we dive into the details of what they mean, let's look at some examples of bad and good file names.
 
-| :x: Bad          | :heavy_check_mark: Good |
+| {fa}`exclamation-triangle,text-error mr-1` Bad          | {fa}`check,text-success mr-1` Good |
```

Comment:
```suggestion
| ❌ Bad          | ✔️ Good |
```

Based on how the [Issue Tracking subchapter](https://the-turing-way.netlify.app/communication/os-comms/os-comms-issue-tracking.html) was written, I think we need to copy/paste the emoji for it to render properly